app/views/devise/registrations/edit.html.erb

```diff
@@ -1,53 +1,97 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col col-md-12 col-lg-8">
+      <div class="card">
+        <section class="card-body">
+          <h2>Edit <%= resource_name.to_s.humanize %></h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+          <%= form_for(resource, as: resource_name,
```

Comment:
Lets move this form into a partial to make this life lighter

Rails partial: https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials